File: 10/Operations.py
import math

def main():
    fn = float(input())
    sn = float(input())

    print(f"""
Square Root of first number: {math.sqrt(fn)}
First number raised to the power of second number: {fn**sn}
Absolute value of first number: {fn}
Absolute value of second number: {sn}
""");

if __name__ == "__main__":
    main()

File: 10/Operations.java
import java.util.*;

public class Operations{
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        float fn = sc.nextFloat();
        float sn = sc.nextFloat();

        System.out.printf("""
            Square Root of first number: %.1f
            First number raised to the power of second number: %.1f
            Absolute value of first number: %.1f
            Absolute value of second number: %.1f
                """,Math.sqrt(fn),Math.pow(fn, sn),fn,sn);
        sc.close();

    }
}
